modules/missenaBidAdapter.js

```diff
@@ -148,18 +148,19 @@ export const spec = {
         payload.consent_string = bidderRequest.gdprConsent.consentString;
         payload.consent_required = bidderRequest.gdprConsent.gdprApplies;
       }
-      const baseUrl = bidRequest.params.baseUrl || ENDPOINT_URL;
-      if (bidRequest.params.test) {
-        payload.test = bidRequest.params.test;
+      const bidParams = bidRequest.params;
+      const baseUrl = bidParams.baseUrl || ENDPOINT_URL;
+      if (bidParams.test) {
+        payload.test = bidParams.test;
       }
-      if (bidRequest.params.placement) {
-        payload.placement = bidRequest.params.placement;
+      if (bidParams.placement) {
+        payload.placement = bidParams.placement;
       }
-      if (bidRequest.params.formats) {
-        payload.formats = bidRequest.params.formats;
+      if (bidParams.formats) {
+        payload.formats = bidParams.formats;
```

Comment:
it seems you are just changing variable names to avoid the detection

Reply:
How can we avoid triggering duplication errors in this case?

Reply:
You should import common functions from a library